src/Package/Impl/DataInspect/VariableGridHost.xaml.cs

```diff
@@ -54,7 +54,7 @@ public partial class VariableGridHost : UserControl {
         internal void SetEvaluation(VariableViewModel wrapper) {
             VsAppShell.Current.AssertIsOnMainThread();
 
-            if (wrapper.TypeName == "NULL" && wrapper.Value == "NULL") {
+            if (wrapper.TypeName == null && wrapper.Value == null) {
```

Comment:
We should probably be checking `TypeName` only then, since it sounds like a null value is generally valid.